android/sound.cpp

```diff
@@ -33,6 +33,7 @@ CSound::CSound ( void           (*fpNewProcessCallback) ( CVector<short>& psData
                  void*          arg,
                  const QString& strMIDISetup,
                  const bool     ,
+                 const QString&,
```

Comment:
It's suboptimal that we have to add implementation-specific flags such as this jack option to all implementations (Windows, Android, Mac), but I can't think of any sane alternatives. Also, it follows the existing logic for jack-specific options (`bNoAutoJackConnect`), so I guess it's ok.